src/node.cc

```diff
@@ -156,6 +156,42 @@ static Isolate* node_isolate = nullptr;
 static v8::Platform* default_platform;
 
 
+static void PrintErrorString(const char *format, ...) {
+  va_list ap;
+  va_start(ap, format);
+#ifdef _WIN32
+  HANDLE hConsoleStderr = GetStdHandle(STD_ERROR_HANDLE);
+  bool isTTY = uv_guess_handle(_fileno(stderr)) == UV_TTY;
```

Comment:
Style issue: `is_tty` and maybe make it `const` while you're here.  `hConsoleStderr` is a little incongruous too.

What is the reason that you use STD_ERROR_HANDLE one line up and stderr here?